telegram/ext/_conversationhandler.py

```diff
@@ -542,6 +557,8 @@ async def _initialize_persistence(
                 'persistence!'
             )
 
+        # Here we will fill the self._conversations variable with conversations from persistence,
+        # and this includes child conversations.
```

Comment:
maybe just include "this handler and it's child conversations" in the docstring?